### PR TITLE
Contact form validation pair program

### PR DESCRIPTION
**Pull request by AkshithaReddy1899**
# What does this PR do?
This PR is raised to add email validation to contact form

# Description of the task to be completed

- Use GitHub workflow
- Configured the project using linters
- Fix linter errors
- Use descriptive PR
- Use Pair Program
- Add email validation and submit the form only when the requirements are fullfilled

-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -318,7 +319,8 @@ <h1 class="about-header header">About me</h1>
             maxlength="500"
           ></textarea>
         </label>
-        <button type="submit" class="project-btn submit btn">
+        <p id="error"></p>
+        <button type="submit" id="submit" class="project-btn submit btn">
           Get in touch
         </button>
```
Comment (oloomoses) [inline index.html:325]: - [ ] email field containing uppercase characters should generate an error as per the project requirements. You can check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string.
Kindly implement this.

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AkshithaReddy1899) [reply index.html:325]: " email field containing uppercase characters should generate an error as per the project requirements." This requirement is fulfilled I kindly suggest you check the project on the local server and not the live link as this branch is not merged in the main branch

**Review by oloomoses — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**